src/views/navbar/components/profileDropdown.js

```diff
@@ -39,9 +43,12 @@ const UserProfileDropdownListItem = styled.li`
   }
 `;
 
-type ProfileProps = Object;
+type ProfileProps = {
+  user: UserInfoType,
+  dispatch: Function,
+};
 
-export const ProfileDropdown = (props: ProfileProps) => {
+export const ProfileDropdown = connect()((props: ProfileProps) => {
```

Comment:
Small nit: we don't use this pattern anywhere else in the codebase. Instead we always export the `connect`ed version:

export default `connect()(ProfileDropdown)`

Could we make that small change? Then let's just ship this and iterate forward, I don't have any other Pro user upsell ideas at the moment

Reply:
Yeah I know but this is already `export const`ed, not `export default`ed, so I didn't want to assign another variable just to do

```JS
const NotConnectedProfileDropdown = () => {}

export const ProfileDropdown = connect()(NotConnectedProfileDropdown);
```

That's even more weird imo? But I don't really care, happy to refactor.

Reply:
Yeah, totally understand - seems like it could be:

```js
const ProfileDropdown = () => {}
export default connect()(ProfileDropdown)
```

right?

Not a big deal either way, but since connect is injecting props it might be helpful to keep the location of that consistent across files for easier debugging or refactoring :)

Reply:
Nope, since `export const ProfileDropdown` will be `import { ProfileDropdown } from 'whatever'` but `export default connect()(ProfileDropdown)` will be `import ProfileDropdown from 'whatever'`.

Those are not the same thing.

Reply:
Totally understand how the imports/exports are different, but I'm suggesting that since we only ever import this into one place (`src/views/navbar/index.js:7`) we can simply export default and then change the import to `import ProfileDropdown` in the `navbar` :)

Reply:
Oh I see, I thought this was used in more places. On it!